doc/eip.md

```diff
@@ -12,28 +12,57 @@ requires: EIP-1014, EIP-7702
-   - It is permissioned: the party that holds the secret key has the ultimate say on which chains will get the CREATE2 factory deployments.
-   - This requires carefully guarding a secret key, if it is exposed or lost, then deployments are no longer guaranteed on new chains.
-   - If the first transaction is not a successful CREATE2 factory deployment, than it is no longer possible to have a CREATE2 factory at the common address; this can happen by human error for example.
-3. Have popular CREATE2 deployment factories deployed on new chains by default. This is, for example, what OP Stack does as part of their [preinstalls](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/libraries/Preinstalls.sol) including the CREATE2 factory contracts mentioned above. This ensures that the CREATE2 factory contracts have known address and code. The downsides to this approach are:
-   - It is not standardized nor adopted by all chains.
-   - It is permissioned as a chain can choose to not include a specific CREATE2 factory contract preinstalled.
-   - Attempts to standardize this with [RIP-7740](https://github.com/ethereum/RIPs/blob/master/RIPS/rip-7740.md) have not been successful.
+### Existing Solutions
 
-This ERC proposes a permissionless alternative fourth mechanisms to the existing ones described above with none of their downsides. Additionally, it standardizes a set of deployment parameters for a **unique** CREATE2 factory deployment. This ensures a common CREATE2 factory for the community, instead of having multiple competing copies at different addresses with slightly different code. This single CREATE2 factory copy can be used to bootstrap additional deterministic deployment infrastructure (such as the aforementioned comprehensive CreateX universal contract deployer).
+There are currently three main approaches to this problem:
+
+#### 1. Nick's method
+
+Use Nick's method to randomly generate a signature for a transaction **without** [EIP-155](./eip-155.md) replay protection that deploys the CREATE2 factory. Nick's method ensures that there is no known private key for an account that deploys the CREATE2 factory, meaning that the resulting contract will have a deterministic address and code on all chains. This strategy is used by [Arachnid/deterministic-deployment-proxy](https://github.com/Arachnid/deterministic-deployment-proxy), one of the most widely used CREATE2 factory contracts.
+
+**Downsides**:
+
+- It does not work on chains that only accept EIP-155 replay-protected transactions.
+- It is sensitive to changes in gas parameters on the target chain since the gas price in the deployment transaction is sealed, and a new one cannot be signed without a private key.
```

Comment:
```suggestion
- It is sensitive to changes in gas parameters on the target chain since the gas price and limit in the deployment transaction is sealed, and a new one cannot be signed without a private key.
```